invokeai/app/invocations/flux_text_encoder.py

```diff
@@ -0,0 +1,81 @@
+from typing import Literal
+
+import torch
+from transformers import CLIPTextModel, CLIPTokenizer, T5EncoderModel, T5Tokenizer
+
+from invokeai.app.invocations.baseinvocation import BaseInvocation, invocation
+from invokeai.app.invocations.fields import FieldDescriptions, Input, InputField
+from invokeai.app.invocations.model import CLIPField, T5EncoderField
+from invokeai.app.invocations.primitives import ConditioningOutput
+from invokeai.app.services.shared.invocation_context import InvocationContext
+from invokeai.backend.flux.modules.conditioner import HFEncoder
+from invokeai.backend.stable_diffusion.diffusion.conditioning_data import ConditioningFieldData, FLUXConditioningInfo
+
+
+@invocation(
+    "flux_text_encoder",
+    title="FLUX Text Encoding",
+    tags=["image", "flux"],
+    category="image",
+    version="1.0.0",
+)
+class FluxTextEncoderInvocation(BaseInvocation):
+    """Encodes and preps a prompt for a flux image."""
+    clip: CLIPField = InputField(
+        title="CLIP",
+        description=FieldDescriptions.clip,
+        input=Input.Connection,
+    )
+    t5_encoder: T5EncoderField = InputField(
+        title="T5Encoder",
+        description=FieldDescriptions.t5Encoder,
+        input=Input.Connection,
+    )
+    max_seq_len: Literal[256, 512] = InputField(description="Max sequence length for the desired flux model")
+    positive_prompt: str = InputField(description="Positive prompt for text-to-image generation.")
```

Comment:
Is this really a positive prompt or is it just any prompt?

Reply:
Both!

Reply:
Ok let's name it `prompt` then

Reply:
[4eca995](https://github.com/invoke-ai/InvokeAI/pull/6739/commits/4eca99574b29465e65663d2d0e4af93ee4dcc7aa)